RESTResource/RESTResource.js

```diff
@@ -337,6 +337,12 @@ export default class RESTResource {
     this.dispatch(this.fetchAction(this.cachedProps));
   }
 
+  hasPerm(state, perms) {
+    const currentPerms = _.get(state, ['okapi', 'currentPerms'], {});
+    const reqPerms = _.isArray(perms) ? perms : perms.split(',');
+    return !reqPerms.find(perm => !currentPerms[perm]);
```

Comment:
This is an elegant line, though I did have to look at it a couple of times before I got it.

Reply:
@MikeTaylor thanks. I tried to find a one liner for this. Not the easiest to read though :/